mz_bokeh_package/authentication/auth.py

```diff
@@ -31,7 +31,7 @@ def get_user(request_handler: RequestHandler) -> bool | None:
     """
 
     # bypass authentication in the case of health.py dashboard to allow GCP to perform health checks
-    if request_handler.request.path.split("/")[-1] in ("health", "error"):
+    if request_handler.request.path.endswith("/health"):
```

Comment:
@captKuk the `get_user` method is a requirement of Bokeh Server, it is used by the Bokeh Server when you provide the `--auth-module` parameter to the `bokeh serve` command (as we do in the `Dockerfile`). See https://docs.bokeh.org/en/2.4.3/docs/user_guide/server.html#auth-module for more info.

Reply:
@oriyudilevich , right... I looked through the Dockerfile but just skimmed over the `auth-module` flag. Then the name is ok. However, I think the docstring of this function should be updated, as it is, it is already outdated. I think something like this would be ok:
```
""" Function required by the bokeh server for authentication of the user. For simplicity it returns `True` for a successfully authenticated user and the required `None` for unknown users. For more details also on the signature of the function see:

https://docs.bokeh.org/en/2.4.3/docs/user_guide/server.html#auth-module
"""
```

Apart from this, the PR is ok.

Reply:
@captKuk - I updated the docstring. I added the link to the top of the module since it's relevant for both methods in the module.